docs/tutorial/masking.html

```diff
@@ -367,8 +367,8 @@
 </script>
 
 Conclusion
-====================================================================================================
```

Comment:
This is a marker to make a title, why do you remove it ?

Reply:
pige pas pourquoi j'ai enlevé ça done